src/state_machine.cpp

```diff
@@ -311,17 +333,6 @@ bool StateMachine::shortcutPush()
   return this->hasState() && this->setCurrentState(this->current_state_->shortcutPush());
 }
 
-bool StateMachine::shortcutDoublePush()
```

Comment:
If you remove this function definition then you should also remove the declaration in the header file.